contracts/vaults/Claim.sol

```diff
@@ -120,7 +135,8 @@ contract Claim is Base {
 
         asset.safeTransfer(claim.beneficiary, claimBounty.hacker);
         asset.safeTransfer(claim.committee, claimBounty.committee);
-        //storing the amount of token which can be swap and burned so it could be swapAndBurn in a separate tx.
+
+        //storing the amount of tokens which can be swapped and burned so it could be swapAndBurn in a separate tx.
```

Comment:
Maybe remove this comment as it is not really stored here anymore but in the registry contract, or maybe say send or approve instead of store